engine/engine/content/builtins/scripts/edn.lua

```diff
@@ -118,53 +121,90 @@ local type_to_primitive_encoder = {
   ["function"] = encode_function,
   ["thread"] = encode_thread
 }
+local metatable_to_primitive_encoder = {
+  [GOScriptInstance] = encode_script,
+  -- [GuiScriptInstance] = encode_script, (it doesn't work for *.gui_script)
```

Comment:
Doesn't work at the moment, `gui_script` doesn't have its values in registry

Reply:
Why is that?

Reply:
I didn't check why, but there is no table with values as it is for script and render_script.
probably gu_ script does it some other way, as many other things =(